client/src/components/PortfolioFilters.tsx

```diff
@@ -449,6 +482,7 @@ function FilterAccordion(props: {
           <summary
             onClick={(e) => {
               e.preventDefault();
+              !isOpen && onOpen && onOpen();
```

Comment:
yeah I guess right now it's provided in every case so could be required - I was just thinking since it's not necessary to the functioning of the component, just an opportunity to do something extra when it opens, it felt like should be optional. Like maybe we decide at some point we don't need to log the open event, or maybe only for new filters as we add them, then we can just not provide the prop rather than remove it from the component.